frotz.py

```diff
@@ -187,28 +188,28 @@ def _paste_cb(self, button):
         self._vte.paste_clipboard()
 
     def __key_press_cb(self, window, event):
-        if event.state & gtk.gdk.CONTROL_MASK and event.state & gtk.gdk.SHIFT_MASK:
+        if event.state & Gtk.gdk.CONTROL_MASK and event.state & Gtk.gdk.SHIFT_MASK:
```

Comment:
This looks like an incomplete port, look for the correct method and make the necessary changes. 
If you're not sure what the correct methods are you can check [here](https://lazka.github.io/pgi-docs/index.html)

Reply:
Thanks I will take a look at these ones.